connexion/decorators/uri_parsing.py

```diff
@@ -188,7 +188,7 @@ def _make_deep_object(k, v):
         root_key = k.split("[", 1)[0]
         if k == root_key:
             return (k, v, False)
-        key_path = re.findall(r'\[(\w+)\]', k)
+        key_path = re.findall(r'\[(\S[^\[\]]+)\]', k)
```

Comment:
Will it work with 1-char values? It looks here like 1 non-spaces, followed by 1+ other chars, giving minimum 2 in total. It was min 1 before the change.

Reply:
```
In [3]: re.match(r'\w+', 'hello')                                                                                     
Out[3]: <re.Match object; span=(0, 5), match='hello'>

In [4]: re.match(r'\w+', 'h')                                                                                         
Out[4]: <re.Match object; span=(0, 1), match='h'>

In [8]: re.match(r'(\S[^\[\]]+)', 'he')                                                                               
Out[8]: <re.Match object; span=(0, 2), match='he'>

In [9]: re.match(r'(\S[^\[\]]+)', 'h')                                                                                

In [10]:                                                   
```

Reply:
let me check

Reply:
adjusted